envisage/tests/test_egg_basket_plugin_manager.py

```diff
@@ -295,7 +276,8 @@ def _create_broken_distribution_eggdir(self, egg_pat, replacement=None):
 
         Parameters
         ----------
-        egg_pat: a glob pattern for the egg in `self.egg_dir` eg 'foo.bar*.egg'
+        egg_pat: a glob pattern for the egg in `self.eggs_dir`
```

Comment:
Drive-by style fix, unrelated to the rest of the PR. (For some reason this was showing up in my local `flake8` runs.)